src/raudio.c

```diff
@@ -1697,6 +1711,24 @@ void SetMusicPitch(Music music, float pitch)
 {
     SetAudioBufferPitch(music.stream.buffer, pitch);
 }
+// Set the music to loop or not
+void SetMusicLooping(Music *music, bool loop)
+{
+    music->looping = loop;
+
+#if defined(SUPPORT_FILEFORMAT_XM)
```

Comment:
I'd prefer to avoid `SetMusicLooping()` and `GetMusicLooping()`, that's the reason to allow `music.looping` direct access, like other data structures. Actually, those two functions are only required for the XM module... I'd prefer some other way to set this, maybe on the `UpdateMusicStream()` call?